benchmarks/benchmark_throughput.py

```diff
@@ -6,13 +6,13 @@
 
 import torch
 
-from hivemind.moe.client import RemoteExpert
-from hivemind.moe.server import ExpertBackend, Server
-from hivemind.moe.server.layers import name_to_block
+import hivemind
+from hivemind import P2P
+from hivemind.dht import DHT
```

Comment:
nit:

```suggestion
```

I'd suggest to:

- Either keep `import hivemind`, use `hivemind.DHT` and `hivemind.P2P` (i.e., apply this suggestion), and keep the "from imports" only for the long module names. This is good because it minimizes the number of imports.
- Or remove `import hivemind` and use the "from imports" only, like it was before this PR. This is good because it minimizes this file's diff.

Mixing both of the mentioned import styles is rather confusing.